{{cookiecutter.project_dirname}}/Makefile

```diff
@@ -99,6 +103,30 @@ remote: pip_update  ## Install remote requirements and dependencies
 report:  ## Run coverage report
 	./scripts/report.sh
 
+.PHONY: runserver
+runserver:  ## Django run
+	python3 -m manage runserver 0:8000
+
+.PHONY: scan_docker_image
+scan_docker_image:  ## Scan docker image vulnerabilities
```

Comment:
remove this one, will be added in separate PR

Reply:
okk